index.html

```diff
@@ -20,7 +20,7 @@ <h1>0data Hello World</h1>
         <ul>
             <li><a href="./fission/">Fission</a></li>
             <li><a href="./remotestorage/">remoteStorage</a></li>
-            <li><a href="./solid/">Solid</a></li>
+            <li><a href="./solid/solid-rest-api/">Solid</a></li>
```

Comment:
You choose to present solid-rest-api as the solid reference. I would not have made this choice

```suggestion
            <li><a href="./solid/">Solid</a></li>
```

Reply:
Well, we can argue which one is the best for beginners, but I think we must choose one. The idea is to provide a simple example for people to get started, if the first thing we show them is a list of examples to choose from, that will confuse them more than anything.

I think the one using the REST API is the best one, because it's the one using the least libraries which was the initial motivation of this hello world. But I guess that's up for debate. As @rosano mentioned, maybe we want to choose the one that is more similar to the other protocols.

Reply:
I've said before that I prefer library versions as the primary one.

Reply:
Ok I've set the `solid-file-client` as the default example.